src/prompts/generateCustomerInfoSQL.ts

```diff
@@ -6,29 +6,27 @@ a call. These are some examples of the tables. Please return with a joined
 query that fetches most relevant information from each table for a given phone
 number.
 
-Parameter name is "strAnlNr" in the facilities table. This is the same as the
-"strAnlnr" in the customer_events table. Put the value of strAnlNr in the
+Parameter name is "strAnlnr" in the facilities table. This is the same as the
+"strAnlnr" in the customer_events table. Put the value of strAnlnr in the
 T-SQL query as a parameter.
 
 Reply in a JSON structure with the T-SQL query  that fetches the relevant information. It is very
 important that your response only includes the valid T-SQL, no comments or other text.
 Do not create aliases for columns, MAKE THE T-SQL AS SHORT AS POSSIBLE.
 Do not list unneccesary columns, only the ones needed!
 
-The reply JSON structure should look like this, there must always be at least one parameter to replace:
-
-IMPORTANT! Do not rename fields! They have to be exactly the same as in the examples below!
+To return less data, consider doing count and aggregation queries, and only return the most relevant data.
 
+The reply JSON structure should look like this:
 \`\`\`
 {
-  "sql": "SELECT ... FROM ... WHERE ... f.strAnlnr='???strAnlNr???' ...",
-  "paramsToReplace": [
-    { "param": "strAnlNr", "placeholder": "???strAnlNr???" }
-  ]
+  "sql": "SELECT ... FROM ... WHERE ... f.strAnlnr='@strAnlnr' ...;",
```

Comment:
Always only have strAnlnr param, so less dynamicism and chances for bugs, and we always replace it with our actual value to get less risk of fetching other users data from example queries.